src/utils/SendETH.sol

```diff
@@ -1,11 +0,0 @@
-// SPDX-License-Identifier: AGPL-3.0
```

Comment:
Why is this deleted? I'm still using it to send ETH to the subdao proxy.

Reply:
Thought it was breaking CI and wasn't used, but turns out all good - added back

Reply:
This is still an empty file.

Reply:
Fixed